src/main/java/bio/terra/service/snapshotbuilder/utils/EnumerateConceptsQueryBuilder.java

```diff
@@ -114,6 +114,23 @@ public Query buildEnumerateConceptsQuery(
         .build();
   }
 
+  public Query getConceptsFromConceptIds(List<Integer> conceptIds) {
+    Concept concept = Concept.asPrimary();
+    FieldVariable nameField = concept.name();
+    FieldVariable conceptId = concept.conceptId();
+    List<SelectExpression> select = List.of(nameField, conceptId);
+
+    return new Query.Builder()
+        .select(select)
+        .tables(List.of(concept))
+        .where(
+            new FunctionFilterVariable(
+                FunctionFilterVariable.FunctionTemplate.IN,
+                conceptId,
+                conceptIds.stream().map(Literal::new).toArray(Literal[]::new)))
```

Comment:
I think I kinda like keeping the `FunctionFilterVariable` from knowing about types beyond `SelectExpression`, and letting it be the caller's problem. Thinking about it, I might think we should maybe explore sublcasses for the different FunctionTemplates if we did that, because `List<Integer>`  would only work for `IN`/`NOT_IN`. We already have some special casing for different templates, it might be work it just from simplifying validation.

Reply:
Instead a constructor, you could add a factory method for `IN`. This would be similar to `BooleanAndOrFilterVariable.and()`:
```
  static FunctionFilterVariable in(FieldVariable fieldVariable, List<Integer> values) {
    return new FunctionFilterVariable(FunctionTemplate.IN, fieldVariable, 
        values.stream().map(Literal::new).toArray(Literal[]::new));
  }
```